Source/Android/app/src/main/java/org/dolphinemu/dolphinemu/features/settings/ui/viewholder/FilePickerViewHolder.java

```diff
@@ -45,15 +46,21 @@ public void bind(SettingsItem item)
     }
     else
     {
-      mTextSettingDescription.setText(NativeLibrary
-              .GetConfig(mFilePicker.getFile(), item.getSection(), item.getKey(),
-                      mFilePicker.getSelectedValue()));
+      mTextSettingDescription.setText(mFilePicker.getSelectedValue(getAdapter().getSettings()));
```

Comment:
These descriptions are always blank.

Reply:
I don't understand your comment. This is the line that makes the selected path show up in the UI.

Reply:
When viewing the Paths submenu, all Paths have a blank description. It should display the value saved for that path instead.

Reply:
It does display the saved value for me. However, it doesn't display anything when you haven't set a path, since the C++ config system's default for all paths is "".

Reply:
We display default paths in QT as of #8710. I also added default paths in #7920 for Android. See `SettingsFragmentPresenter.getDefaultNANDRootPath` and the code after that. Hopefully I implemented them properly.

Reply:
Ah, so DolphinQt doesn't just display the value straight from the config system... I suppose I could implement something similar in Android without actually changing what the config system considers to be the default value.

Reply:
Should be fixed with the commit I just added.

Reply:
It is. Apparently I can't close my conversations on other people's PRs. That's annoying.

Reply:
Huh, I thought that was possible... I'll mark it as resolved myself, then.